lib/components/avatar-ai/index.tsx

```diff
@@ -0,0 +1,11 @@
+import Logo from "@appflowy-chat/assets/logo.svg?react";
```

Comment:
import {ReactComponent as Logo} from '@appflowy-chat/assets/logo.svg'

Reply:
Should I use such an import format every time I import an SVG icon as a React Component?

Reply:
@qinluhe Does it relate to different version of vite-plugin-svgr?  should we use vite-plugin-svgr 4.3.0? 
in the latest version of vite-plugin-svgr, it uses this syntax: ```
import Logo from "@appflowy-chat/assets/logo.svg?react";
Member
```

Reply:
so it's ok, ignore me

Reply:
@qinluhe Make sure when you import it there is no version conflict